packages/flutter_test/lib/src/window.dart

```diff
@@ -610,16 +619,36 @@ class FakeAccessibilityFeatures implements ui.AccessibilityFeatures {
 ///     testing and mocking purposes.
 class TestPlatformDispatcher implements ui.PlatformDispatcher {
   /// Constructs a [TestPlatformDispatcher] that defers all behavior to the given
-  /// [dart:ui.PlatformDispatcher] unless explicitly overridden for test purposes.
+  /// [ui.PlatformDispatcher] unless explicitly overridden for test purposes.
   TestPlatformDispatcher({
     required ui.PlatformDispatcher platformDispatcher,
-  }) : _platformDispatcher = platformDispatcher;
+  }) : _platformDispatcher = platformDispatcher {
+    _configuration = _platformDispatcher.configuration.copyWith();
+    _updateViews();
+  }
 
-  /// The [dart:ui.PlatformDispatcher] that is wrapped by this [TestPlatformDispatcher].
+  /// The [ui.PlatformDispatcher] that is wrapped by this [TestPlatformDispatcher].
   final ui.PlatformDispatcher _platformDispatcher;
 
   @override
-  ui.VoidCallback? get onMetricsChanged => _platformDispatcher.onMetricsChanged;
+  TestFlutterView? get implicitView {
+    return _platformDispatcher.implicitView != null ?
+      _testViews[_platformDispatcher.implicitView!.viewId]! :
+      null;
```

Comment:
Checks some other examples in the code base, but I believe the canonical formatting of this is:

```suggestion
    return _platformDispatcher.implicitView != null
      ? _testViews[_platformDispatcher.implicitView!.viewId]!
      : null;
```

Reply:
Huh... Booleans are the opposite way with the operators at the end... That's a bit frustrating. But I can do that, it's usually my preference anyways.

Reply:
Done.